java/pending_code_changes.txt

```diff
@@ -1,6 +1,21 @@
 GitHub project changes:
  - Changed tag to vX.Y.Z from libphonenumber-X.Y.Z; this may affect ports and
    derived projects.
+Metadata structure changes:
+ - Mobile and Fixed-Line blocks are no longer assumed to inherit missing data
+   from the GeneralDesc, but are treated like every other phone number type.
+   This means that for the non-geographical country codes, like +800, the
+   example number has been moved from generalDesc to the relevant number types,
+   and the code in getExampleNumberForNonGeoEntity has been changed to look at
+   these sub-types for an example number.
+   This also means that the "NA" and "-1" blocks present in the metadata to
+   indicate that no mobile or fixed-line numbers appear for the entity have been
+   removed.
+   There should no longer be an exampleNumber at the generalDesc level, but it
+   should be present at every PhoneNumberDesc with data.
+Metadata changes
```

Comment:
I don't see a commit in master with an IR change since the last release? https://github.com/googlei18n/libphonenumber/commits/master

If it's a pending pull request let's not merge until this PR is in. I say this PR is priority, and we merge it before any other open PRs at this point; since it's be a pain to sync with the base branch this otherwise.

In case I'm missing something and there is an IR change:
We'll have to merge this manually anyway, but it's less likely to be missed or have formatting errors by the merger if it's last (after code changes) since that's where the rest of the metadata changes will show.

And since you're using the proper format anyway, optionally add a ':' after changes :)